### ISSUE TITLE
composer.json refers non-existance package

### ISSUE THREAD

**Issue by pokehanai** [FlorianWolters/PHP-Component-Util-Reflection#1]
It looks `composer.json` needs to be updated, to refer to newer version of your another package. :)

```
$ ./composer.phar install
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for florianwolters/component-util-reflection dev-master -> satisfiable by florianwolters/component-util-reflection[dev-master].
    - florianwolters/component-util-reflection dev-master requires florianwolters/component-core-stringutils 0.2.*@beta -> no matching package found.
```

**Comment by FlorianWolters** [comment]
Thank you for your report. I will fix it on the weekend.

**Comment by pokehanai** [comment]
Okay.

**Comment by FlorianWolters** [comment]
Should be fixed with commit 4e484e36e8536d6c952b6c2ffe6d72d31a526e68 in release v0.3.0.
